### PR TITLE
Create ui-c8y-1019-15-2-fix-log-output-when-running-local-dev-server-wrong-ports-5881.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-57344](https://cumulocity.atlassian.net/browse/MTM-57344)
Original PR: [5983](https://github.softwareag.com/IOTA/cumulocity-ui/pull/5983)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1019-15-2-fix-log-output-when-running-local-dev-server-wrong-ports-5881.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fix log output when running local dev server (wrong ports) (#5881) [GRAFT][release/cd] (#5983)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1019-15-2-fix-log-output-when-running-local-dev-server-wrong-ports-5881.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Fix wrong port numbers in log output when running local development server
```

File: content/change-logs/application-enablement/ui-c8y-1019-15-2-fix-log-output-when-running-local-dev-server-wrong-ports-5881.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57344
+version: 1019.15.2
+---
+Fix log output when running local dev server (wrong ports) (#5881) [GRAFT][release/cd] (#5983)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1019-15-2-fix-log-output-when-running-local-dev-server-wrong-ports-5881.md:17]: The Claude AI suggested to use the following body:
```suggestion
When running a local development server, the log output incorrectly displayed the port numbers as 8080 and 8443. This has been fixed to show the correct port numbers that the local development server is actually using, which are port 80 for HTTP and port 443 for HTTPS. This improves the developer experience by avoiding confusion about which ports are being used.
```

**Review by janhommes — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1019-15-2-fix-log-output-when-running-local-dev-server-wrong-ports-5881.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57344
+version: 1019.15.2
+---
+When running a local development server with the `ng serve` command, the log output incorrectly displays the port numbers if the default port is already used. This has been fixed to show the correct port numbers that the local development server is actually using. This improves the developer experience by avoiding confusion about which ports are being used.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1019-15-2-fix-log-output-when-running-local-dev-server-wrong-ports-5881.md:17]:
```suggestion
When running a local development server with the `ng serve` command, the log output incorrectly displayed the port numbers if the default port was already used. This has been fixed to show the correct port numbers that the local development server is actually using. This improves the developer experience by avoiding confusion about which ports are being used.
```

File: content/change-logs/application-enablement/ui-c8y-1019-15-2-fix-log-output-when-running-local-dev-server-wrong-ports-5881.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fix wrong port numbers in log output when running local development server
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1019-15-2-fix-log-output-when-running-local-dev-server-wrong-ports-5881.md:3]:
```suggestion
title: Correct port numbers shown in log output when running local development server
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**